rest-server/src/models/job.js

```diff
@@ -118,7 +118,7 @@ class Job {
 
   putJob(name, data, next) {
     if (!data.outputDir.trim()) {
-      data.outputDir = `${launcherConfig.hdfsUri}/Output/${name}`;
+      data.outputDir = `${launcherConfig.hdfsUri}/Output/${data.username}/${name}`;
```

Comment:
>{data.username [](start = 58, length = 14)

to clarify, even we fix the name here, we still need to fix launcher, otherwise, different user still cannot use same job name, right?

Reply:
right